modules/dxp/apps/saml/saml-test/src/testIntegration/java/com/liferay/saml/internal/servlet/filter/test/SpSessionTerminationSamlPortalFilterTest.java

```diff
@@ -117,4 +123,7 @@ private SafeCloseable _setSamlEnabledWithSafeCloseable(
 	@Inject
 	private static CounterLocalService _counterLocalService;
```

Comment:
Hey @manuelecastro, I know that it is not related to your changes here. But can you make the `_counterLocalService`  a non-static field?

Thx